src/SkyCoords.jl

```diff
@@ -7,12 +7,74 @@ export AbstractSkyCoords,
        ICRSCoords,
        GalCoords,
        FK5Coords,
-       separation
-
-import Base: convert
+       separation,
+       str2rad
```

Comment:
Do you think it's important to export this function?

Reply:
One use case I can think of wanting to load in a text file with string coordinates and end up with a table of their numerical equivalents. Based on that use-case, though, it makes more sense to put this method somewhere in AstroBase.

Given that, what do you say we leave this unexported and consider its usage in a different module?

Reply:
Your use case is good, of course, but I'm not sure it'll be so frequent to beg for this function to be exported.  I think it can always be imported if really necessary.